docs/sources/mimir/release-notes/v2.7.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "Grafana Mimir version 2.7 release notes"
+menuTitle: "V2.7 release notes"
+description: "Release notes for Grafana Mimir version 2.7"
+weight: 300
+---
+
+# Grafana Mimir version 2.7 release notes
+
+Grafana Labs is excited to announce version 2.7 of Grafana Mimir.
+
+The highlights that follow include the top features, enhancements, and bugfixes in this release. For the complete list of changes, see the [changelog](https://github.com/grafana/mimir/blob/main/CHANGELOG.md).
+
+## Features and enhancements
+
+- **Support for `keep_firing_for` option to ruler configuration**
+- **Experimental query sharding improvements**:
+  A new configuration parameter, `-query-frontend.query-sharding-target-series-per-shard`, allows query sharding to take into account cardinality of similar requests executed previously when computing the maximum number of shards to use.
+- **Experimental support for native histogram ingestion**:
+  Native histograms can now be ingested. The new per-tenant limit `-ingester.native-histograms-ingestion-enabled` controls whether native histograms are stored or ignored.
+
+### Alertmanager improvements
+
+- **New metrics** The following upstream metrics are now exposed:
+  - `cortex_alertmanager_dispatcher_aggregation_groups`
+  - `cortex_alertmanager_dispatcher_alert_processing_duration_seconds`
+
+### Helm chart improvements
+
+The Grafana Mimir and Grafana Enterprise Metrics Helm chart is now released independently. See the [corresponding documentation](/docs/helm-charts/mimir-distributed/latest/) for more information.
+
+## Important changes
+
+In Grafana Mimir 2.7, the default vaules of the following configuration options have changed:
+
+- `-blocks-storage.bucket-store.batch-series-size` is now enabled by default with a value of `5000`.
```

Comment:
WDYT about mentioning this in the `Features and enhancements`? It's a fairly big change in the implementation and we've seen reduction in OOMs and memory usage for large requests. We've previously mentioned this as the "store-gateway streaming series results"

Reply:
Nice catch, I'm going to put this at the top of the new features and enhancements.